collect_app/src/main/AndroidManifest.xml

```diff
@@ -270,7 +270,7 @@ the specific language governing permissions and limitations under the License.
 
         <!-- Enable Shortcuts for Command Actions -->
         <activity
-            android:name=".activities.AndroidShortcuts"
+            android:name=".activities.AndroidShortcutsActivity"
```

Comment:
As far as  I understand it's OK to change this but let me know if that's not the case!